### PR TITLE
Fix empty landscape

### PR DESCRIPTION
**Pull request by vauxhall**

### CODE REVIEW

File: Client/Frontend/Library/HistoryPanel.swift [hunk 1]
```diff
@@ -197,6 +205,8 @@ class HistoryPanel: SiteTableViewController, LibraryPanel {
             return deferMaybe(result)
         }
     }
+
```
Comment (ecotopian) [inline Client/Frontend/Library/HistoryPanel.swift:208]: are these needed? :)

**Review by ecotopian — APPROVED** [review]
Looks great!

File: Client/Frontend/Library/HistoryPanel.swift (hunk 1 above)
Comment (vauxhall) [reply Client/Frontend/Library/HistoryPanel.swift:208]: Good observation, removed :)

Comment (vauxhall) [inline Client/Frontend/Library/HistoryPanel.swift:208]:
```suggestion
```

File: Client/Frontend/Library/HistoryPanel.swift [hunk 2]
```diff
@@ -197,6 +205,7 @@ class HistoryPanel: SiteTableViewController, LibraryPanel {
...
```
Comment (vauxhall) [inline Client/Frontend/Library/HistoryPanel.swift:209]:
```suggestion
```

**Review by ecotopian — APPROVED** [review]

**Pull request merged**